src/loaders/ttf/tvgTtfReader.cpp

```diff
@@ -389,10 +389,11 @@ uint32_t TtfReader::glyph(uint32_t codepoint, TtfGlyphMetrics& gmetrics)
     auto glyph = this->glyph(codepoint);
     if (glyph == INVALID_GLYPH) {
         TVGERR("TTF", "invalid glyph id, codepoint(0x%x)", codepoint);
-        return INVALID_GLYPH;
```

Comment:
@lpogic It doesn't seem necessary to change the `glyph()` function. I'd like to ask you to retain the original code unless this is needed for your fix. Could you please double-check this?

Reply:
This change is not essential and was just to make debugging easier. I will revert it soon.

Reply:
Coud you please apply this code?
```
if (glyph == INVALID_GLYPH || !glyphMetrics(glyph, gmetrics)) {
        TVGERR("TTF", "invalid glyph id, codepoint(0x%x)", codepoint);
        return INVALID_GLYPH;
}
return glyph;
```